code-challenges.js

```diff
@@ -47,3 +103,24 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+// Goal is to create a function that takes a set of any two numbers and outputs the difference with the smaller number being subtracted from the larger number.
+// Create a function called bigMinusSmall
+// bigMinusSmall will take in two parameters: num1, num2
+// Set up a conditional statement for if a num1 is greater than num2, log the output of num1 - num2.
+// Set up an else if statement underneath the aforementioned if statement stating that if num1 < num2, then log the put of num2 - num1.
+// Set up an else statment that will log the string "Please enter different numerical values" because if the parameters did not meet the conditions of the 1st if statement and the 2nd else if statment, that means that num1 is equal to num2.
+// Outside of the function, log the output of bigMinusSmall(number1, number2) and bigMinusSmall(number3, number4) to confirm that the smaller number has been subtracted from the larger number.
+// So I actually ran this code, and it outputs the expected result for Set One and Set Two but it also outputs an 'undefined' after subtracing the smaller num from the bigger num, and I'm not really sure why. 
+
+const bigMinusSmall = (num1, num2) => {
+  if(num1 > num2){
+    console.log(num1 - num2)
+  }
+  else if(num1 < num2){
+    console.log(num2 - num1)
+  } else{
+    console.log("Please enter different numerical values")
```

Comment:
When writing functions it is a best practice to keep your `console.log` on the invocation of the function and not the inner logic. If you removed the logs you may find that your functions returns undefined.